cpp/log/cert.cc

```diff
@@ -583,8 +583,8 @@ util::Status Cert::OctetStringExtensionData(int extension_nid,
   // Callers don't care whether extension is missing or invalid as they
   // usually call this method after confirming it to be present.
   const StatusOr<void*> ext_struct = ExtensionStructure(extension_nid);
-  if (!ext_struct.ok()
-      && ext_struct.status().CanonicalCode() == Code::NOT_FOUND) {
+  if (!ext_struct.ok() &&
```

Comment:
I prefer the old way, but only by a small margin (with old one, operator is more obvious, with new one, nicer alignment), so I didn't might it too much. If there's a `clang-format` option to do it, sure, why not.